websites/G/Granblue Fantasy/presence.ts

```diff
@@ -23,13 +77,30 @@ presence.on("UpdateData", async () => {
   } else if (href.includes("/#result"))
     presenceData.details = "In a Quest result screen";
   else if (href.includes("/#raid") || href.includes("/#raid_multi")) {
+    if (health === 0) {
+      presenceData.state = `At ${
+        document.getElementsByClassName(
+          "btn-enemy-gauge prt-enemy-percent alive"
+        )[0].textContent
+      }`;
+    } else if (health === 1) {
+      const boss = gameStatus.boss.param.find(x => x.alive),
+        hp = parseInt(boss.hp);
+      presenceData.state = `${hp.toLocaleString()} [${(
+        (100 * hp) /
+        parseInt(boss.hpmax)
+      ).toLocaleString(void 0, {
+        maximumFractionDigits: 2
+      })}%]`;
```

Comment:
Are you going to refactor this?

Reply:
Yes but I can't do it this week. If it's possible, I'd prefer if this part can be neglected for now and I'll make a followup PR addressing it if I'm successful.

Reply:
Done